docs/paper/dev/api/how-do-plugins-work.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: /dev/how-do-plugins-work
+description: How plugins work in Paper.
+---
+
+# How Plugins Work
+
+## Introduction
+
+Plugins are a way to extend the functionality of a Minecraft server. They are written in JVM-based languages such as 
+Java, Kotlin, Groovy or Scala. Plugins are loaded from the `plugins` folder in the server directory. Plugins will be 
+loaded from a `.jar` file. 
```

Comment:
maybe add something about extending javaplugin and a big note about not using the ctor